### PR TITLE
Breaking change - new tests, invocation type

### PR DESCRIPTION
**Pull request by RichardWright**

### CODE REVIEW

**Comment by RichardWright** [comment]
@ArnoldZokas Want to go through this today?

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
-  - "4"
-  - "5"
-  - "6"
+  - '0.10'
```
Comment (ArnoldZokas) [inline .travis.yml]: let's target only 4 and 6

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+4.6.0
```
Comment (ArnoldZokas) [inline .nvmrc]: let's change this to 4

File: tests/hapi-hobknob-client-tests.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                },
+                expose: function(key, value){
+                    this.plugins['hapi-hobknob-client'][key] = value;
+                }
+            };
+
+            const Hobknob = {
+              etcdPort: 4001,
+              cacheIntervalMs: 30000,
+              etcdHost: 'hobknob-etcd-qa.otenv.com'
```
Comment (ArnoldZokas) [inline tests/hapi-hobknob-client-tests.js]: don't use internal DNS in public repos

File: package.json [hunk 4]
```diff
@@ -20,12 +22,15 @@
...
+    "eslint": "^3.3.1",
+    "eslint-config-defaults": "^9.0.0",
+    "expect.js": "^0.3.1",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.0.2",
+    "rewire": "^2.5.2"
   },
   "dependencies": {
+    "hobknob-client-nodejs": "^2.0.0",
+    "lodash": "^4.17.2"
```
Comment (ArnoldZokas) [inline package.json]: there is only one usage of lodash in the entire project, kind of overkill to pull it in IMHO

File: .nvmrc [hunk 5]
```diff
@@ -1 +1 @@
-4.6.0
+4.0
```
Comment (ArnoldZokas) [inline .nvmrc]: just 4

base because 4 means "use latest 4.x.x", but 4.0 means use "4.0.x"

**Comment by RichardWright** [comment]
@ArnoldZokas Could we get this in?

**Comment by RichardWright** [comment]
@ArnoldZokas

**Comment by RichardWright** [comment]
@ArnoldZokas This pull request is up to date now. Could you review?

**Comment by ArnoldZokas** [comment]
@RichardWright quite a lot of changes in this release. Need to test this on a low-impact system.

**Comment by RichardWright** [comment]
@ArnoldZokas Hermes? Or something even lower?

**Comment by ArnoldZokas** [comment]
Hermes sounds good.

**Comment by RichardWright** [comment]
@ArnoldZokas I can't merge, do you have permissions?

**Pull request merged**